src/services/identity/UsersService.ts

```diff
@@ -63,20 +63,37 @@ class UsersService {
     await this.repository.update(user, { where: { githubId } })
   }
 
-  async findOrCreate(githubId: string | undefined) {
-    const [user] = await this.repository.findOrCreate({
-      where: { githubId },
+  async findOrCreate(githubId: string | undefined): Promise<User> {
```

Comment:
Woah, I just realised we arent using this function anywhere in the codebase apart from our tests ._. 
If you can double confirm this is true, could we remove this method as a whole? This method does not handle undefined githubID gracefully as well, seems to be doing what the `login` function is doing anyways

Reply:
you're right; i'll delete this method and the associated tests. odd that this got added w/o any call sites but ok